packages/kolibri-components/src/KResponsiveWindowMixin.js

```diff
@@ -109,11 +109,17 @@ export default {
         If that issue ever gets addressed, we should make them computed props.
       */
       windowBreakpoint: undefined,
+      windowIsPortrait: false,
+      windowIsLandscape: false,
```

Comment:
FYI you'll likely encounter a conflict with a change I made: https://github.com/learningequality/kolibri/pull/6000/files#diff-60e4b3c76101fb0363a4af6b85fb8719R118

Should be an easy resolution, let me know if you need help

Reply:
Alright, thanks for the heads up